src/main.jsx

```diff
@@ -6,10 +6,10 @@ import 'bootstrap/dist/css/bootstrap.min.css'
 import 'react-multi-carousel/lib/styles.css';
```

Comment:
please move these to the place where they are needed
```js
import 'bootstrap/dist/css/bootstrap.min.css'
import 'react-multi-carousel/lib/styles.css';
```

Reply:
done, move to App.jsx